cmd/kube-controller-manager/app/plugins.go

```diff
@@ -107,7 +107,7 @@ func ProbeExpandableVolumePlugins(config componentconfig.VolumeConfiguration) []
 	allPlugins = append(allPlugins, scaleio.ProbeVolumePlugins()...)
 	allPlugins = append(allPlugins, storageos.ProbeVolumePlugins()...)
 	allPlugins = append(allPlugins, fc.ProbeVolumePlugins()...)
-	if !utilfeature.DefaultFeatureGate.Enabled(features.CSIPersistentVolume) {
+	if utilfeature.DefaultFeatureGate.Enabled(features.CSIPersistentVolume) {
```

Comment:
@NickrenREN Actually that entire `if-block` should be removed since `CSIPlugin` does not implement `ExpandableVolumePlugin` interface.

Reply:
got it, thanks, done